docs/pod_network_outages.md

```diff
@@ -0,0 +1,15 @@
+### Pod outage
```

Comment:
s/pod outage/pod network outage

Reply:
Also let's add this scenario to the list -https://github.com/redhat-chaos/krkn#kubernetesopenshift-chaos-scenarios-supported